<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_dialog</class>
 <widget class="QDialog" name="search_dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>6</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>鹅山通</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="150,618">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	color: black;
	border: 2px solid #33373E;
	border-radius: 10px; 
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="618,200">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="routeLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: #FFFFFF;     
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190);
}</string>
        </property>
        <property name="text">
         <string>重新搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#scrollArea {
	border-radius: 10px;
	border: 2px solid #33373E;
	border-style: solid;
	padding: 10px;
}
QScrollBar:vertical {
border: none;
width: 10px;
margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
background: Gainsboro;
min-height: 20px;
border-radius: 5px;
border: none;
}
QScrollBar::add-line:vertical {
border: 0px solid grey;
height: 0px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
border: 0px solid grey;
height: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
width: 0px;
height: 0px;
}
QScrollBar::handle:vertical:hover,QScrollBar::handle:vertical:pressed{
background: rgb(209, 209, 209);
min-height: 20px;
border-radius: 5px;
border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>558</width>
        <height>198</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
    background-color: none;      /* 背景颜色为白色 */
}</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
